tests/acceptance/FrontendTest.php

```diff
@@ -6,6 +6,46 @@
 class FrontendTest extends oxAcceptanceTestCase
 {
     /**
+     * Adds configuration data for testing
+     */
+    protected function setUp()
+    {
+        parent::setUp();
+
+        $configData = $this->getAmazonPaySettings();
+        if (is_array($configData) && count($configData)) {
```

Comment:
Since we now expect an array we can use `$configData !== []` which is much better than `count`. Count isn't good if we just want to check if the array is empty or not.